dockstore-webservice/src/main/java/io/dockstore/webservice/resources/DockerRepoResource.java

```diff
@@ -761,7 +761,7 @@ public Tool publish(@ApiParam(hidden = true) @Parameter(hidden = true, name = "u
     @ApiOperation(value = "List all published tools.", tags = {
         "containers"}, notes = "NO authentication", response = Tool.class, responseContainer = "List")
     public List<Tool> allPublishedContainers(
-        @ApiParam(value = "Start index of paging. Pagination results can be based on numbers or other values chosen by the registry implementor (for example, SHA values). If this exceeds the current result set return an empty set.  If not specified in the request, this will start at the beginning of the results.") @QueryParam("offset") String offset,
+        @ApiParam(value = "Start index of paging. If not specified in the request, this will start at the beginning of the results.") @DefaultValue("0") @QueryParam("offset") Integer offset,
```

Comment:
Does `@DefaultValue("0")` work to fill in 0 if not provided or is it just informational to the user (I don't actually know this, might be worth a test)

Reply:
To answer Denis's question, per https://docs.oracle.com/javaee/7/api/javax/ws/rs/DefaultValue.html `The default value is used if the corresponding meta-data is not present in the request.` It should provide the value by default if not passed. Is this sufficient proof?

Reply:
I'm a little more confident, but it would be nice to try this anyway above